oaknational/Oak-Web-Application
# Feat/pupil 889/quiz results

## Description

Music year: 2017

### Job Story

**When** I am rendering the review page,
**I want to render the detailed quiz results for a full quiz**
**So that users can see how to improve**.

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2674--oak-web-application.netlify.thenational.academy
2. complete a lesson
3. You should see all quiz results in the review section of lesson, on the quiz bar use the dropdown "results"

<img width="1166" alt="Screenshot 2024-08-13 at 10 11 36" src="https://github.com/user-attachments/assets/0c6bea25-31eb-4db4-a9fb-9c76192d5188">

## Screenshots

How it should now look:
[{screenshots}](https://www.figma.com/design/G7z5ufo2T0CHk14CkadeuB/Pupil-Lesson?node-id=9206-51077&m=dev)

you can also see an example in storybook:
https://deploy-preview-2674--owa-storybook.netlify.thenational.academy/?path=/story/components-pupilcomponents-quizresults--default

## Checklist
